mscore/musescore.cpp

```diff
@@ -6267,6 +6278,11 @@ int main(int argc, char* av[])
       if (settings.value("mixerVisible", false).toBool())
             mscore->showMixer(true);
 
+      QAction* a = new QAction(getAction("palette-shortcut"));
+      QKeySequence keys = QKeySequence(Qt::Key_T);
```

Comment:
Ok... it's "Save As...". Bad idea. Anyway, this should go in shortcuts.xml. We will find a shortcut.